conans/test/unittests/tools/google/test_bazeldeps.py

```diff
@@ -220,6 +220,7 @@ def test_bazeldeps_shared_library_interface_buildfiles():
     includes=["include"],
     visibility=["//visibility:public"],
     deps = [
+        #donotsort
```

Comment:
Is this on purpose missing all spaces in the comment ?

Reply:
Oh, thanks, no, it's a mistake. I'll fix that